app/commands/get_user_review_request_metrics.rb

```diff
@@ -0,0 +1,38 @@
+class GetUserReviewRequestMetrics < PowerTypes::Command.new(:github_user, :limit_month)
+  def perform
+    entry_count = 0
+    total_time = 0
+    review_requests_dic = Hash.new
+    get_valid_pull_requests.each do |pr|
+      if review_req = pr_has_review_requests(pr)
+        time_delta = (review_req.created_at - pr.gh_created_at).to_i
+        unless time_delta > 1.week
+          entry_count += 1
+          total_time += time_delta
+          review_requests_dic[pr.id] = get_pr_info(pr, review_req, time_delta)
+        end
+      end
+    end
+    review_requests_dic["personal_mean"] = entry_count.positive? ? total_time / entry_count : 0
+    review_requests_dic
+  end
+
+  def pr_has_review_requests(pull_request)
+    monkey = GithubUser.find_by(login: "monkeyci")
+    review_requests = pull_request.pull_request_review_requests
+                                  .where.not(github_user_id: monkey.id)
+                                  .order(created_at: :asc)
+    review_requests[0] || nil
+  end
+
+  def get_valid_pull_requests
+    limit_date = (Time.zone.today - @limit_month.to_i.months).beginning_of_day
+    @github_user.pull_requests.where('gh_created_at > ?', limit_date).where.not(merged_by_id: nil)
+  end
+
+  def get_pr_info(pr, review_req, time_delta)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Naming/MethodParameterName: Method parameter must be at least 3 characters long.